app/main.py

```diff
@@ -1,6 +1,20 @@
-from typing import Callable
+from typing import Callable, Any
 
 
-def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+def cache(func: Callable) -> Any:
+    cache_storage = {}
+
+    # use one return
+    def wrapper(*args, **kwargs) -> Any:
+        key = (args, tuple(sorted(kwargs.items())))
+
+        if key in cache_storage:
+            print("Getting from cache")
+            return cache_storage[key]
```

Comment:
delete this line

Reply:
this line `return cache_storage[key]` I literally wrote comment below this line.